easybuild/easyblocks/p/pytorch.py

```diff
@@ -477,9 +477,16 @@ def test_step(self):
         if failed_test_suites != all_failed_test_suites:
             failure_report = ['Failed tests (suites/files):'] + failure_report
             # Test suites where we didn't match a specific regexp and hence likely didn't count the failures
-            failure_report.extend('+ %s' % t for t in sorted(all_failed_test_suites - failed_test_suites))
+            uncounted_test_suites = all_failed_test_suites - failed_test_suites
+            if uncounted_test_suites:
+                failure_report.append('Could not count failed tests for the following test suites/files:')
+                failure_report.extend(sorted(uncounted_test_suites))
```

Comment:
I think (after listing the uncounted test suites) we could add a more explicit explanation and instruction. Something like:

"This will always result in a failure of the test step, because we do not know how many tests have failed, and thus cannot compare to the `max_failed_tests` set in the EasyConfig. Please investigate the cause of failures in these test suite. If you are _sure_ they are harmless, you can consider running with `--ignore-test-failures` to make the test step pass."

Reply:
I did think about adding something like "This will always result in a failure of the test step" but decided for a more clear `raise EasyBuildError('Failing because the PyTorch EasyBlock needs updating!\n' + msg)`

Especially due to the 500 char limit printed by EB I wanted to keep the important information very short. (The full/detailed message is appended)

However I like the instructions for users. So maybe below use: `raise EasyBuildError('Failing because the test accounting in the PyTorch EasyBlock needs updating!\nYou can check the test failures (in the log) manually and if they are harmless, use --ignore-test-failures to make the test step pass.\n' + msg)`

And update & move the comment there (line 501/508) to "We can't be sure how many tests failed so comparing to max_failed_tests cannot reasonably be done"